entries/jQuery.parseHTML.xml

```diff
@@ -15,8 +15,9 @@
     </argument>
   </signature>
   <longdesc>
-    <p><code>jQuery.parseHTML</code> uses a native DOM element creation function to convert the string to a set of DOM elements, which can then be inserted into the document.</p>
+    <p><code>jQuery.parseHTML</code> uses a native functionality to convert the string to a set of DOM nodes, which can then be inserted into the document.</p>
```

Comment:
@arthurvr, @dmethvin: What do you  think about changing "uses a native functionality" to "uses native DOM methods"?